File: 240418/굉장한 숫자/amazing-number.js
const fs = require('fs');
const n = Number(fs.readFileSync(0).toString());

function isUbelievable(n){
    return isOddAndMultipleOfThree(n) || isEvenAndMultipleOfFive(n);
}

function isOddAndMultipleOfThree(n){
    return n % 2 != 0 && n % 3 == 0;
}

function isEvenAndMultipleOfFive(n){
    return n % 2 == 0 && n % 5 == 0;
}

console.log(isUbelievable(n));

File: 240418/굉장한 숫자/amazing-number.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();

        System.out.println(isUbelievable(n));
        
    }

    public static boolean isUbelievable(int n){
        return isOddAndMultipleOfThree(n) || isEvenAndMultipleOfFive(n);
    }

    public static boolean isOddAndMultipleOfThree(int n){
        return n % 2 != 0 && n % 3 == 0;
    }
   
    public static boolean isEvenAndMultipleOfFive(int n){
        return n % 2 == 0 && n % 5 == 0;
    }
}